irohad/ametsuchi/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ add_library(ametsuchi
     impl/in_memory_block_storage_factory.cpp
     impl/flat_file_block_storage.cpp
     impl/flat_file_block_storage_factory.cpp
+    impl/postgres_block_storage.cpp
```

Comment:
I think it is a good time for starting a separation of Ametsuchi target, better to make separate targets for block_storage pg and flat file implementations.